staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -123,7 +123,7 @@ func newStore(c *clientv3.Client, codec runtime.Codec, newFunc func() runtime.Ob
 		pathPrefix:          pathPrefix,
 		groupResource:       groupResource,
 		groupResourceString: groupResource.String(),
-		watcher:             newWatcher(c, codec, groupResource, newFunc, versioner),
+		watcher:             newWatcher(c, codec, groupResource, newFunc, versioner, pagingEnabled),
```

Comment:
I wouldn't pass this argument here for two reasons:
1) we are planning to graduate pagination to stable in 1.29 anyway
2) as described in the bug, without pagination it may be impossible even to list stuff due to gRPC limits...